frontend/src/components/ShootDetails/ShootLifecycleCard.vue

```diff
@@ -16,7 +16,13 @@ SPDX-License-Identifier: Apache-2.0
         </v-list-item-icon>
         <v-list-item-content>
           <v-list-item-title>Hibernation</v-list-item-title>
-          <v-list-item-subtitle class="d-flex align-center pt-1">
+          <v-list-item-subtitle class="d-flex pt-1">
+            <hibernation-constraint-warning
+              :isHibernationPossible="isHibernationPossible"
+              :hibernationPossibleMessage="hibernationPossibleMessage"
+              :hasHibernationSchedules="shootHibernationSchedules.length"
```

Comment:
HTML properties must be `kebap-case` https://vuejs.org/v2/guide/components-props.html#Prop-Casing-camelCase-vs-kebab-case